dtale/views.py

```diff
@@ -1061,7 +1056,7 @@ def view_network(data_id=None):
     :type data_id: str
     :return: HTML
     """
-    if data_id is None or data_id not in global_state.get_data().keys():
+    if data_id is None or global_state.get_data_inst(data_id) is None:
```

Comment:
`global_state.contains` god looking at the original code I wish I did it.  My apologies

Reply:
added

Reply:
```suggestion
    if global_state.contains(data_id):
```

Reply:
if not global_state.contains(data_id): ?